melodies_monet/util/satellite_utilities.py

```diff
@@ -140,11 +140,12 @@ def omps_l3_daily_o3_pairing(model_data,obs_data,ozone_ppbv_varname):
     grid_adjust = xe.Regridder(model_data[['latitude','longitude']],obs_data[['latitude','longitude']],'bilinear')
     mod_col_obsgrid = grid_adjust(column)
     # Aggregate time-step to daily means
-    daily_mean = mod_col_obsgrid.groupby('time.date').mean(numeric_only=True).compute()
-    
+    #daily_mean = mod_col_obsgrid.groupby('time.date').mean().compute()
+    daily_mean = mod_col_obsgrid.resample(time='1D').mean()
```

Comment:
@blychs should there be `numeric_only=True` here?

Reply:
And @mbruckner-work just curious why the 91ad0c3111b8b042e9c92efb359b42bec41e15e5 change? Shouldn't `.groupby('time.date').mean()` and `.resample(time='1D').mean()` give the same result?

Reply:
@zmoon The data is in xarray here. Putting in 'numeric_only=True' causes the code to fail.

Reply:
I believe that @mbruckner-work found that it didn't work, since it is a Dataset instead of a DataFrame, and needed to be taken out.

Reply:
The https://github.com/NOAA-CSL/MELODIES-MONET/commit/91ad0c3111b8b042e9c92efb359b42bec41e15e5 change was done because of the time format of the paired dataset. '.groupby('time.date').mean()' changes the time format to Timestamp while '.resample(time='1D').mean()' keeps the format as datetime64. 
The old way resulted in an error with the time filtering in the plotting code in my most recent tests, hence the change.

Reply:
Thanks @mbruckner-work . Just wanted to make sure we are keeping up with @blychs efforts to supports pandas 2. You could remove the commented out lines that (no longer?) work.

Reply:
It would be great to remove those lines before merging this.

Reply:
Thanks @mbruckner-work appreciate it